Database/Corrections/wotlkNPCFixes.lua

```diff
@@ -984,6 +984,10 @@ function QuestieWotlkNpcFixes:Load()
             [npcKeys.questStarts] = {13500},
             [npcKeys.questEnds] = {13500},
         },
+        [33288] = {
+            [npcKeys.spawns] = {[zoneIDs.ULDUAR]={{-1,-1}},[zoneIDs.THE_PRISON_OF_YOGG_SARON]={{68,40.2}}},
```

Comment:
I am not sure if these spawns should be the other way around in order to work 🤔 I have something in mind that a loop stops when encountering a `-1,-1` spawn.

Reply:
They were showing just fine when i was tweaking them, but i'll change the order